app/gimlet/app.toml

```diff
@@ -162,6 +172,15 @@ uses = ["quadspi"]
 interrupts = {92 = 1}
 task-slots = ["gpio_driver", "rcc_driver"]
 
+[tasks.sensor]
+path = "../../task/sensor"
+name = "task-sensor"
+features = ["itm"]
+priority = 3
+requires = {flash = 16384, ram = 2048 }
+stacksize = 2000
```

Comment:
This seems wrong -- that leaves 48 bytes of RAM for all sensor data?

Reply:
That really *does* seem wrong; investigating.

Reply:
And of course, this is because the sensor data is currently stored on the stack. Adding a comment!